src/components/Dashboard/TodoList.vue

```diff
@@ -1246,7 +1242,7 @@ export default class TodoList extends Mixins(
     const line2 = this.nameRequest
       ? 'You will be returned to your Manage Businesses dashboard where you can use the Name Request ' +
         'associated with this application for a future application.'
-      : 'You will be returned to the Business Registry home page.'
+      : 'You will be returned to the BC Registries home page.'
```

Comment:
@severinbeauvais which page does the user get returned to here where it says "BC Registries home page?

Reply:
This is after deleting a draft IA. (Without the draft IA, they have nothing, hence this message.)

Reply:
Answer: it depends.
1. If they are deleting a draft named IA, they go back to the Manage Business page.
2. If they are deleted a draft numbered IA, they go back to the BCROS home URL (eg, "Start a Benefit Company and Keep
Cooperatives Records up to date").

Reply:
note to self - may need to break out into 2 messages

Reply:
@severinbeauvais 
1. If they are deleting a draft named IA, they go back to the Manage Business page: message should say "You will be returned to My Business Registry."
2. If they are deleted a draft numbered IA, they go back to the business marketing homepage (eg "Start a : message should say "You will be returned to the Business Registry page."